appveyor.yml

```diff
@@ -15,6 +15,11 @@ environment:
     - PYTHON: "C:\\Python34-x64"
     - PYTHON: "C:\\Python35-x64"
 
+init:
+    - ps: Set-WinSystemLocale zh-TW
+    - ps: Start-Sleep -s 5
+    - ps: Restart-Computer
+
```

Comment:
What is going on here? Why would we run our test suite with a specific locale? If we were running the test suite with both English and Chinese locales, that would be one thing, but this change means that we never run the tests in an English locale.